### PR TITLE
Add swoole support.

### PR DESCRIPTION
**Pull request by jmjoy**
Swoole is a popular coroutine library in PHP, so it is better to support swoole.

### CODE REVIEW

File: src/plugin/mod.rs [hunk 1]
```diff
@@ -20,14 +20,13 @@ mod plugin_swoole;
 use crate::execute::{AfterExecuteHook, BeforeExecuteHook};
 use once_cell::sync::Lazy;
 
-// Register plugins here.
+/// Register plugins here.
```
Comment (heyanlong) [inline src/plugin/mod.rs]:
```suggestion
// Register plugins here.
```

**Review by heyanlong — APPROVED** [review]
LGTM

File: src/plugin/mod.rs (hunk 1 above)
Comment (jmjoy) [reply src/plugin/mod.rs]: /// means rust document.

**Comment by huangdijia** [comment]
Coroutine\Http\Client
 Coroutine\MySQL
 Swoole\Coroutine\Http\Client
 Coroutine\Redis

Is there a plan to support them?

**Comment by jmjoy** [comment]
> Coroutine\Http\Client Coroutine\MySQL Swoole\Coroutine\Http\Client Coroutine\Redis
> 
> Is there a plan to support them?

Yes, but not now.

**Comment by heyanlong** [comment]
@huangdijia Please to use [discussions](https://github.com/apache/skywalking/discussions) to discussions instead of here.

**Comment by huangdijia** [comment]
OK

**Pull request merged**